docs/syntax-reference.rst

```diff
@@ -440,6 +440,28 @@ The WITH clause supports the following properties:
     :start-line: 2
     :end-line: 6
 
+.. _insert-into:
+
+INSERT INTO
+-----------
+
+**Synopsis**
+
+.. code:: sql
+
+    INSERT INTO stream_name
+      SELECT select_expr [., ...]
+      FROM from_item
```

Comment:
`from_item` can be either a stream or table, correct?

Reply:
Only `stream` since we decided to not support `table` for now. @rodesai you may want to clarify this.

Reply:
If it is only from stream -> into stream, then the syntax example should be updated/clarified.

Like:

```
    INSERT INTO stream_name
      SELECT select_expr [., ...]
      FROM from_stream
...
```